config/vufind/OpenIDConnectClient.ini

```diff
@@ -0,0 +1,22 @@
+; Open ID Connect authentication settings; see the [Authentication] section of config.ini to enable.
+[OpenIDConnect]
```

Comment:
How about renaming this to [Default]? This would improve forward-compatibility if we introduce multiple IdP support.

Reply:
It does not hurt, so I changed it